### PR TITLE
Split in two packages: the frontend and a JS library

### PR DESCRIPTION
**Pull request by wraitii**

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -26,6 +26,18 @@
...
+        "src/**/*.tsx",
+        "src/**/*.vue",
+        "../hyle-js/src/blocks.ts",
+        "../hyle-js/src/erc20.test.ts",
+        "../hyle-js/src/erc20.ts",
+        "../hyle-js/src/network.ts",
+        "../hyle-js/src/transactions.ts",
+        "../hyle-js/src/cosmos.ts",
+        "../hyle-js/src/utils.ts",
+        "../hyle-js/src/indexer.ts"
```
Comment (wraitii) [inline tsconfig.json:40]: bruh

File: src/explorer/Home.vue [hunk 2]
```diff
@@ -58,3 +57,4 @@ const transactions = computed(() => Object.values(transactionData));
 </template>
 
 <style scoped></style>
+../../../hyle-js/src/blocks../../../hyle-js/src/transactions
```
Comment (hhalex) [inline src/explorer/Home.vue:60]: ?

File: src/explorer/RegisterContract.vue [hunk 3]
```diff
@@ -111,3 +112,4 @@ const submitForm = async () => {
         </div>
     </div>
 </template>
+../../../hyle-js/src/network
```
Comment (hhalex) [inline src/explorer/RegisterContract.vue:115]: ?

File: src/explorer/TransactionView.vue [hunk 4]
```diff
@@ -51,3 +52,4 @@ const parsedTx = computed(() => getParsedTx(txData.value));
         </div>
     </div>
 </template>
+../../../hyle-js/src/transactions../../../hyle-js/src/proto/tx
```
Comment (hhalex) [inline src/explorer/TransactionView.vue:55]: ?

**Review by hhalex — APPROVED** [review]

**Pull request merged**